_posts/en/newsletters/2024-04-03-newsletter.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Bitcoin Optech Newsletter #296'
+permalink: /en/newsletters/2024/04/03/
+name: 2024-04-03-newsletter
+slug: 2024-04-03-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Revisiting consensus cleanup:** Antoine Poinsot [posted][poinsot
+  cleanup] to Delving Bitcoin about revisiting Matt Corallo's [consensus
+  cleanup][topic consensus cleanup] proposal fom 2019 (see [Newsletter
+  #36][news36 cleanup]).  He starts by attempting to quantify the worst
+  case of several problems the proposal could fix, including: the
+  ability to create a block that can take more than 3 minutes to verify
+  on a modern laptop and 90 minutes to verify on a Raspberry Pi 4; the
+  ability for miners to steal subsidy and make LN insecure using the
+  [time warp attack][topic time warp] with about a month of
+  preparation; trick light clients into accepting fake transactions
```

Comment:
Maybe? 

```suggestion
  preparation; tricking light clients into accepting fake transactions
```

Reply:
Alternatively, it seems to me that there is at least one word missing here.

```suggestion
  preparation; the ability to trick light clients into accepting fake transactions
```